src/chatClient.cpp

```diff
@@ -1504,11 +1504,6 @@ void Client::onConnStateChange(presenced::Client::ConnState /*state*/)
 
 void Client::terminate(bool deleteDb)
 {
-#ifndef KARERE_DISABLE_WEBRTC
```

Comment:
I think we should keep this call here, since the `terminate()` can be call for different reasons than a logout, and in all cases the calls should be hang up.